prisma/schema.prisma

```diff
@@ -30,9 +30,8 @@ model User {
 
   createdAt DateTime  @default(now())
   updatedAt DateTime? @updatedAt()
-  deletedAt DateTime?
```

Comment:
i think we need deletedAt if this was added by binod dai. @saroj-rumsan  can you confirm with binod dai